MTL-1919 boot-order workaround (#2278)
Cray-HPE/docs-csm: scripts/workarounds/boot-order/README.md

> # Boot-Order Workaround

Reviewer: ```suggestion
# Boot Order Workaround
```

   Uploading new metal-lib.sh to ncn-m002:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-m003:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-s001:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-s002:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-s003:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-s004:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-w001:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-w002:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-w003:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Uploading new metal-lib.sh to ncn-w004:/srv/cray/scripts/metal/metal-lib.sh ... Done
   Refreshing the bootorder on [11] NCNs ... Done
   The following NCNs contain the boot-order patch:
   ncn-m001
   ncn-m002
   ncn-m003
   ncn-s001
   ncn-s002
   ncn-s003
   ncn-s004
   ncn-w001
   ncn-w002
   ncn-w003
   ncn-w004
   This workaround has completed.
   ```

> ## Origin

Reviewer: Is this useful/relevant information for our customer-facing CSM product documentation?

Author: Script is copied directly from 1.3/1.4, so I want the diff to be 0 and have a link to the origin like the kdump workaround.

> The original script and library used live in [metal-provision](https://github.com/Cray-HPE/metal-provision/tree/v1.2.2):

Reviewer: ```suggestion
This script originated from the [metal-provision repository](https://github.com/Cray-HPE/metal-provision/tree/v1.2.2):
```

> The original script and library used live in [metal-provision](https://github.com/Cray-HPE/metal-provision/tree/v1.2.2):

Reviewer: Is this useful information for customers?
